tests/integration/ceph_mgr_test.go

```diff
@@ -113,9 +113,21 @@ func (s *CephMgrSuite) TearDownSuite() {
 	s.installer.UninstallRook()
 }
 
-func (s *CephMgrSuite) execute(command []string) (error, string) {
-	orchCommand := append([]string{"orch"}, command...)
-	return s.installer.Execute("ceph", orchCommand, s.namespace)
+func (s *CephMgrSuite) execute(command []string, maxRetries int) (error, string) {
+	tries := 0
+	orchestratorCommand := append([]string{"orch"}, command...)
+	for {
+		err, output := s.installer.Execute("ceph", orchestratorCommand, s.namespace)
+		tries++
+		if err == nil || maxRetries == 1{
```

Comment:
```suggestion
		if err == nil || maxRetries == 1 {
```

not sure how this is passing go fmt...
When should `maxRetries` be one? Why not always retry?

Reply:
Also, the logic is a bit backward. In Go it's more comment to check `err != nil` instead of `err == nil`, if you could re-adapt the change based on that.

Reply:
`maxRetries = 1` when I do not want to use the _retries system_, for example , inside the loop waiting for the orchestrator module. (I think that it is needed to have the possibility of executing a command only one time , independently of the result of the command, which can fail, and the fail can be expected).

By other comments in other prs I thought that this was a good way, and I found it quite readable, but i have changed it following your advice. Thanks